relay/exchange/order.py

```diff
@@ -138,11 +139,11 @@ def __init__(
                          expiration_timestamp_in_sec,
                          salt,
                          v=0,
-                         r=b'',
-                         s=b'')
+                         r=hexbytes.HexBytes('0x00'),
+                         s=hexbytes.HexBytes('0x00'))
```

Comment:
r and b have been empty before, now they contain the zero byte AFAICT. Was that intended?